### PR TITLE
Upgrade contracts to cw-plus 0.13.2

### PR DESCRIPTION
**Pull request by orkunkl**
- Upgrade cw-plus to 0.13.2
- Upgrade cw-std to 1.0.0-beta8

### CODE REVIEW

File: contracts/cw20-atomic-swap/Cargo.toml [hunk 1]
```diff
@@ -18,16 +18,16 @@ backtraces = ["cosmwasm-std/backtraces"]
...
-cw-utils = { version = "0.12.1" }
-cw2 = { version = "0.12.1" }
-cw20 = { version = "0.12.1" }
-cosmwasm-std = { version = "1.0.0-beta5" }
-cw-storage-plus = { version = "0.12.1" }
+cw-utils = { version = "0.13.2" }
+cw2 = { version = "0.13.2" }
+cw20 = { version = "0.13.2" }
+cosmwasm-std = { version = "1.0.0-beta8" }
+cw-storage-plus = { version = "0.13.2" }
```
Comment (ueco-jb) [inline contracts/cw20-atomic-swap/Cargo.toml:25]: You could simplify this for consistency.
```suggestion
cw-utils = "0.13.2"
cw2 = "0.13.2"
cw20 = "0.13.2"
cosmwasm-std = "1.0.0-beta8"
cw-storage-plus = "0.13.2"
```
in other Cargo.toml files as well.

File: contracts/cw20-streams/src/contract.rs [hunk 2]
```diff
@@ -320,8 +320,8 @@ mod tests {
         let info = mock_info("cw20", &[]);
         instantiate(deps.as_mut(), mock_env(), info.clone(), msg).unwrap();
 
-        let sender = Addr::unchecked("Alice").to_string();
-        let recipient = Addr::unchecked("Bob").to_string();
+        let sender = Addr::unchecked("alice").to_string();
+        let recipient = Addr::unchecked("bob").to_string();
```
Comment (ueco-jb) [inline contracts/cw20-streams/src/contract.rs:324]: Those changes were unnecessary?

**Review by ueco-jb — APPROVED** [review]
I'd rather have update and release in separate PRs, in case update breaks something - easier to revert.

File: contracts/cw20-streams/src/contract.rs (hunk 2 above)
Comment (orkunkl) [reply contracts/cw20-streams/src/contract.rs:324]: It is necessary after address normalization bug fix.

Comment (ueco-jb) [reply contracts/cw20-streams/src/contract.rs:324]: Ah cool, forgot about it. :+1:

**Comment by orkunkl** [comment]
Did this CI error occur on cw-plus before @ueco-jb?

**Comment by ueco-jb** [comment]
Discussed via Discord - deploy key needed to be reset, CI works fine now.

**Pull request merged**